### DISCUSSION TITLE
⛔ Use of grave accent in code (instead of single quote)

### DISCUSSION THREAD

**Discussion by tamelander** [elementor/elementor#22046] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi,
I have a ton of warnings in Google Chrome Dev Console about deprecated something. The Dev Console warnings are presented like this:

*** Edit, I can't get the editor to show backticks, so I use an acronym: -BKT-  ***

-BKT-elementor.helpers.getUniqueID()-BKT- is hard deprecated since 3.0.0 - Use -BKT-elementorCommon.helpers.getUniqueId()-BKT- instead

 Just curious, why is the code in dev-tools.min.js using grave accent instead of single quotes inside double quotes?  Is it PHP Execution Operator backticks?
 https://www.php.net/manual/en/language.operators.execution.php

Example Code in dev-tools.min.js
(G.default.deprecated("requireContainer", "3.7.0", "Extend -BKT-$e.modules.editor.CommandContainerBase-BKT- or -BKT-$e.modules.editor.CommandContainerInternalBase-BKT-"),
                        !T.container && !T.containers)

### Describe the solution you'd like

I am just being curious. Only Elementor warnings use backticks.

**Comment by nicholaszein** [comment]
Hello @tamelander.

This space is for submitting Feature Request, not to report bugs. Please be mindful of our [Contribution Guidelines](https://elemn.to/gh-contributing) when submitting your issues and discussions.

As for your question, this is how the browser shows the warning safely, it's not what it's included in the code. And this warning specifically only appears when used by incompatible add-ons who still use deprecated code.

Thank you for contributing. 🙏 

Kind regards
